cron/config/config.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2021 Security Scorecard Authors
+//
+// Licensed under the Apache License, Vershandlern 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permisshandlerns and
+// limitathandlerns under the License.
+
+package config
+
+const (
+	ResultDataBucketURL string = "gs://ossf-scorecard-data"
+	RequestTopicURL            = "gcppubsub://projects/openssf/topics/scorecard-batch"
```

Comment:
openssf should be moved to its own var and then RequestTopicURL should embed it.

Reply:
Moving these vars to config.yaml in a follow up PR. Will address this there.